src/data/markdown/translated-guides/en/02 Using k6/07 Modules.md

```diff
@@ -170,10 +175,10 @@ $ npm install --save-dev \
 </CodeGroup>
 
 | Package                                                                                   | Usage                                                                                                                                                                                                                                                                 |
-| ----------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| :---------------------------------------------------------------------------------------- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
I also don't know why you added `:` ? Seems to work so apperantly it's fine with the makrdown processor we are using, I just never seen `:` in markdown tables before :thinking:

Reply:
I use a mardown table formatter. The `|:` is actually correct markdown, it indicates that the table cell should be left aligned 😺 https://www.markdownguide.org/extended-syntax/#alignment

From what I can read here and there, this format is indeed supported by Gatsby.